Dockerfile

```diff
@@ -1,34 +1,36 @@
 ARG TURING_API_IMAGE
+FROM bash:alpine3.14 as swagger-ui
+
+WORKDIR /app
+
+ARG openapi_spec_file="/app/swagger-ui/openapi.bundle.yaml"
+
+# Build swagger ui deps
+COPY ./scripts/swagger-ui-generator /app/swagger-ui-generator
+
+RUN cd /app/swagger-ui-generator && ./swagger-ui-generator.sh \
+    --spec-file ${openapi_spec_file} \
+    --output /app/swagger-ui
+
 FROM ${TURING_API_IMAGE}
 
 ARG TURING_USER=${TURING_USER:-turing}
 ARG TURING_USER_GROUP=${TURING_USER_GROUP:-app}
 
-ENV TURINGUICONFIG_SERVINGDIRECTORY "/app/turing-ui"
+ARG openapi_spec_file=/app/swagger-ui/openapi.bundle.yaml
+ARG turing_ui_dist_path=ui/build
 
-USER root
-RUN apk add --no-cache bash
```

Comment:
`bash` is still required, because this image uses a custom entrypoint, that requires bash shell

Reply:
Interesting, ok

Reply:
I expected it to be reproducible when you tested it locally

Reply:
i used /bin/sh with my mounted config

Reply:
Hmm doesn't seem like it, I'm removing this bash requirement: https://github.com/ashwinath/turing/blob/d929942aa09f03fc78c206167c9ca1135af9e2a3/infra/charts/turing/templates/deployment.yaml#L65-L74

Reply:
Entrypoint is defined inside the Dockerfile. Args you posted are passed into it as arguments:
https://github.com/gojek/turing/blob/70fb3a0b63aac2dd9d3f8d972588c9eedb16eda9/Dockerfile#L34

Reply:
ok, I missed this part. the way i tested it was like this `docker run --net host --rm -ti -v ${PWD}:/stuff --entrypoint /bin/sh asia.gcr.io/gods-dev/ashwin-hack/turing:15`. Which was why i missed it out.